pkgs/applications/misc/bemenu/default.nix

```diff
@@ -0,0 +1,45 @@
+{ stdenv, fetchFromGitHub, cairo, cmake, libxkbcommon
+, pango, fribidi, harfbuzz, pcre, pkgconfig
+, ncursesSupport ? true, ncurses ? null
```

Comment:
Is this necessary since nix is lazy?

Reply:
I was thinking about this too, however is there some good way to declare x11 support since it depends on two different group of packages (`x11` and `xorg`)?

The rest I just put to maintain the same way.

Reply:
Checked this and don't see the need for the `? null` though I could be wrong

Reply:
my first comment was aimed at the `? null` bit and not the `ncursesSupport` bit if that wasn't clear as I'm not sure what you mean